server/authentication/clientCredentials.test.ts

```diff
@@ -1,7 +1,11 @@
 import generateOauthClientToken from './clientCredentials'
 
 describe('generateOauthClientToken', () => {
-  it('Token can be generated', () => {
+  it('Token can be generated using default client id and secret', () => {
```

Comment:
(Unrelated to code change - just added to increase code coverage and make the pipeline happy)